test/mocha-babel.js

```diff
@@ -10,5 +10,7 @@ function areGeneratorsSupported() {
 }
 
 if (!areGeneratorsSupported()) {
-	require('babel/register');
+	require('babel/register')({
+		ignore: /index\.js|lib\/|node_modules/
+	});
```

Comment:
The separate ES5 sanity check is no longer necessary. This simply prevents babel from instrumenting our production files. I did test it by adding an unused generator function to `index.js` and in one of the `lib` files. That caused syntax errors in Node 0.10